saleor/urls.py

```diff
@@ -17,6 +19,11 @@
         digital_product,
         name="digital-product",
     ),
+    path(
+        "download-invoice/<uuid:invoice_hash>/",
```

Comment:
Backstory behind this is that we have a plugin that generates an invoice file on our side (saves to local storage to be compliant with people using S3 for ex.) Plugin saves the invoice then it needs to generate an URL and save it on Invoice model (this is a flow when our built-in plugin handles the creation). It can generate such URL due to having this view so we can serve it later. Then an email mutation can be trigerred and the mail is being sent with URL already saved on model.
Another options we have instead of having that view:
- combine it with existing view `digital_content` so that is our only file-serving endpoint
- re-phrase it to more generic view that can serve content for all plugins for example if there will be need.
- drop the endpoint at all and serve somehow else?

Please advise, especially @patrys since I didn't explain it properly to you, hope this clarifies what needs to be accomplished.

Reply:
Django media files have a `url` property that returns a public URL of the file. We should not act as a proxy here.